client/src/featureform/register.py

```diff
@@ -1591,6 +1603,142 @@ class Customer:
+class MultiFeatureColumnResource(ColumnResource):
+    def __init__(
+        self,
+        dataset: SourceVariant,
+        df: pd.DataFrame,
+        entity_column: Union[Entity, str] = "",
+        variant: str = "",
+        owner: str = "",
+        inference_store: Union[str, OnlineProvider, FileStoreProvider] = "",
+        timestamp_column: str = "",
+        include_columns: List[str] = None,
+        exclude_columns: List[str] = None,
+        description: str = "",
+        schedule: str = "",
+        tags: List[str] = None,
+        properties: Dict[str, str] = None,
+    ):
+        """
+        Registering multiple features from the same table.
+
+        **Example**
+        ```
+        @ff.entity
+        class Customer:
+        # Register multiple columns from a dataset as features
+            transaction_features = ff.MultiFeature(
+                dataset,
+                dataframe_name,
+                variant="quickstart",
+                inference_store=redis,
+                entity_column="CustomerID",
+                timestamp_column="Timestamp",
+                exclude_columns=["TransactionID", "IsFraud"],
+                inference_store=redis,
+            )
+        ```
+
+        Args:
+            dataset (SourceVariant): The dataset to register features from
+            df (pd.DataFrame): The client.dataframe to register features from
+            entity_column (Union[Entity, str]): The name of the column in the source to be used as the entity
+            variant (str): An optional variant name for the feature.
+            inference_store (Union[str, OnlineProvider, FileStoreProvider]): Where to store for online serving.
+            include_columns (List[str]): List of columns to be registered as features
+            exclude_columns (List[str]): List of columns to be excluded from registration. Cannot be used with include_columns
+        """
+        self.include_columns = include_columns or []
+        self.exclude_columns = exclude_columns or []
+        self.tags = tags or []
+        self.properties = properties or {}
+        self.owner = owner
+        self.description = description
+        self.schedule = schedule
+        self._resources = []
+        register_columns = self._get_feature_columns(
+            df, include_columns, exclude_columns, entity_column, timestamp_column
+        )
+        self._create_feature_columns(
+            df,
+            dataset,
+            register_columns,
+            entity_column,
+            timestamp_column,
+            inference_store,
+            variant,
+        )
+
+    def _create_feature_columns(
+        self,
+        df,
+        dataset,
+        register_columns,
+        entity_column,
+        timestamp_column,
+        inference_store,
+        variant,
+    ):
+        df_has_quotes = False
+        for column_name in df.columns:
+            if '"' in column_name:
```

Comment:
why dont we just do this inside of the modify_quotes function?

Reply:
Oh nevermind, we dont do this in modify_quotes because we need to check if the dataframe columns have quotes, not the columns in register_columns